app/web/service/notifications.ts

```diff
@@ -1,11 +1,10 @@
 import { Empty } from "google-protobuf/google/protobuf/empty_pb";
 import {
   GetNotificationSettingsReq,
-  RegisterPushNotificationReq,
+  RegisterPushNotificationSubscriptionReq,
```

Comment:
It looks like the build step from GitLab is saying this isn't properly exported?